### DISCUSSION TITLE
Use a production WSGI server instead?

### DISCUSSION THREAD

**Discussion by Peppershaker** [apache/airflow#18519] [Q&A] [answered]
I'm running a single machine instance of Airflow 2.1.4 with LocalExecutor on a RDS Postgres 13.

When I invoke the scheduler on the command line with `airflow scheduler`, it tells me the Flask application isn't running using a production WSGI server
```
* Serving Flask app "airflow.utils.serve_logs" (lazy loading)
* Environment: production
* WARNING: This is a development server. Do not use it in a production deployment.
* Use a production WSGI server instead.
* Debug mode: off 
```

However, the messages is missing from log files if running the scheduler as a systemd service.

Do I need to worry about the warning msg?

**Comment by potiuk** [comment, marked as answer]
It's harmless. It's an internal server run by the executor to share logs with the webserver. It's been already corrected in `main` to use 'production` setup (thought it's not REALLY) needed i this case as the log "traffic" and characteristics is not production-webserver like.
The fix will be released in Airflow 2.2 (~ month from now) .

**Reply by Peppershaker** [reply]
Thank you very much!
